resources/macos_user.rb

```diff
@@ -36,7 +45,7 @@ def admin_user?
 
 action :create do
   execute "add user #{new_resource.username}" do
-    command "#{sysadminctl} -addUser #{new_resource.username} -password #{new_resource.password} #{admin_user?}"
+    command "#{sysadminctl} -addUser #{new_resource.username} #{full_name?} -password #{new_resource.password} #{admin_user?}"
```

Comment:
Per another comment of mine, let's pass an array of arguments here so the `command` method can shell-escape all the strings for us.

```ruby
command [sysadminctl, '-addUser', new_resources.username, full_name, '-password', new_resource.password, admin_user?]
```